src/sha256.c

```diff
@@ -45,8 +45,13 @@ void sha256_transform(SHA256_CTX *ctx, const BYTE data[])
 {
 	WORD a, b, c, d, e, f, g, h, i, j, t1, t2, m[64];
 
-	for (i = 0, j = 0; i < 16; ++i, j += 4)
-		m[i] = (data[j] << 24) | (data[j + 1] << 16) | (data[j + 2] << 8) | (data[j + 3]);
```

Comment:
Left shift of unsigned char promotes variable to "int". Variable can become negative in that case(UB), further shift is UB etc.

Reply:
let's discuss this after we conclude the discussion in rdbLoadIntegerObject